src/main/java/jun/yu/maven/test/PowerMockito/Human.java

```diff
@@ -7,6 +7,7 @@
     public String address2;
     public String address3;
     public String address;
+    private String address4;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "address4" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)